Dockerfile

```diff
@@ -12,28 +12,22 @@ RUN (curl -sL https://rpm.nodesource.com/setup_10.x | bash) && \
 # Install container dependencies:
 RUN yum -y install epel-release && \
     yum -y update && \
-    yum -y install make \
-    yum -y install git \
-    yum -y m4 \
-    yum -y zlib-devel \
-    yum -y install redis
+    yum -y install make git m4 zlib-devel redis
 
 RUN yum -y groupinstall "Development Tools"
```

Comment:
In order to maximize the space saving benefits, I'd recommend combining all the yum commands into one single `RUN` instruction.

Reply:
Is `groupinstall` considered a different command? I thought it was, which would have required a separate invocation